Add Kruskal's minimum spanning tree algorithm implementation.

izanbf1803 opened the pull request:
Tested, working correctly.

ankit167 commented on cache/lru_cache.py:
```diff
@@ -0,0 +1,56 @@
+      self.value = v
+      self.prev = None
+      self.next = None
+
```
2 blank spaces between classes

ankit167 commented on cache/lru_cache.py:
```diff
@@ -0,0 +1,56 @@
+        self.capacity = capacity
+        self.head = LRUCacheNode(None, 'head')
+        self.tail = LRUCacheNode(None, 'tail')
+        self.head.next = self.tail # tail being most recent
```
Atleast two spaces before inline comment. Line 13,14

ankit167 commented on cache/lru_cache.py:
```diff
@@ -0,0 +1,56 @@
+        self.tail.prev = self.head # head being oldest
+        self.data = {}
+
+    def deleteNode(self,node):
```
Add space after ','. Line 17, 24, 41, 50

ankit167 replied:
Snake case recommended for function/class name

ankit167 commented on cache/lru_cache.py:
```diff
@@ -0,0 +1,56 @@
+        last.next = newNode
+        self.tail.prev = newNode
+        newNode.next = self.tail
+        newNode.prev = last
```
Please add sample test case inside main()

ankit167 commented on cache/lru_cache.py:
```diff
@@ -0,0 +1,56 @@
+class LRUCacheNode:
```
Please add brief overview of the LRU implementation. It will be easier for folks to understand the code.

izanbf1803 commented:
I deleted my implementation of the LRU Cache because i don't have experience with that DS.
I also implemented the Kruskal algorithm for the minimum spanning tree, and i have explained my implementation (common implementation).

keon commented:
@izanbf1803 Thanks!
We might have to consolidate the comment style but that could go onto other PR.  :)
